### PR TITLE
fix some bug

### PR DESCRIPTION
**Pull request by FE-Friday**
1、[bugfix: ToolTips组件在鼠标经过显示后，鼠标滚轮滚动页面，ToolTips不会消失](https://github.com/TencentBlueKing/bkui-vue2/commit/513b9afcf064cf504813325f25ae7a3ed91db9a5)
2、[bugfix: 解决value是字符串时，allowHtml属性找不到报错问题](https://github.com/TencentBlueKing/bkui-vue2/commit/d824015b4f452ca93541eb0383994bb9a4ba8603)

### CODE REVIEW

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/bkui-vue2?pullRequest=376) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/bkui-vue2?pullRequest=376) before we can accept your contribution.<br/>**0** out of **2** committers have signed the CLA.<br/><br/>:x: 墨香阁主<br/>:x: FE-Friday<br/><hr/>**墨香阁主** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/bkui-vue2?pullRequest=376) it.</sub>

File: src/directives/tooltips.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ const defaultOptions = {
   interactive: true,
   boundary: 'window',
   content: '',
-  allowHTML: true,
+  allowHTML: false,
```
Comment (ielgnaw) [inline src/directives/tooltips.js:51]: 这个地方的默认值还是需要为 `true`。

这里默认值为 `true` 或者 `false` 其实都会造成之前部分用户需要修改，我们统计过，改为 `true` 的修改成本相对较小~

File: src/utils/util.js [hunk 2]
```diff
@@ -358,3 +358,64 @@ export function checkOverflow (el) {
...
 }
+
+/**
+ * 监听鼠标滚轮事件
+ * @param {Dom Object} elem 目标dom
+ * @param {callback} callback 回调函数
+ * @param {Dom Object} useCapture 是否冒泡
+ */
+
+ export function addWheelListener (elem, callback, useCapture) {
```
Comment (ielgnaw) [inline src/utils/util.js:369]: 这一行前面多了一个空格，导致代码规范扫描不通过~

**Comment by ielgnaw** [comment]
已 merge

**Pull request closed without merging**